hardhat.config.js

```diff
@@ -86,27 +86,38 @@ for (const f of fs.readdirSync(path.join(__dirname, 'hardhat'))) {
 const withOptimizations =
   argv.enableGasReport || argv.compileMode === 'production';
 
+
+if (Config.useFaucet){
+  console.log(`Use faucet: ${Config.faucetUrl}`)
+} else {
+  console.log(`Skipping faucet requests: USE_FAUCET set to false`);
+}
+
 const requestFaucet = async (wallet, amount) => {
   if (!Config.useFaucet) {
-    console.log(`Skipping faucet request: USE_FAUCET set to false`);
     return;
   }
-  console.log('Requesting faucet...');
   const data = { amount: amount, wallet: wallet };
-  console.log(`URL: ${Config.faucetUrl}`);
   if (Config.faucetUrl.length === 0) {
     console.log('Unable to request faucet: FAUCET_URL is empty');
     return;
   }
-  console.log(`Wallet = ${data.wallet}, amount = ${data.amount}`);
-  try {
-    const result = await axios.post(Config.faucetUrl, data);
-  } catch (err) {
-    console.log(`Failed to send request to faucet: ${err}`);
-  }
+  console.log(`Request '${data.amount}' tokens for wallet ${data.wallet} from faucet`);
+
+    try {
+      await axios.post(Config.faucetUrl, data);
```

Comment:
Ivan also asked for displaying (at least, or checking) status code like [here](https://github.com/axios/axios):
![image](https://user-images.githubusercontent.com/2947151/147739873-a5fd2ebc-6e44-480b-86eb-05ecbcc7ef99.png)

Reply:
`axios.post(x, y)
    .then((response) => {
        console.log(response.status);
        console.log(response.data);
});`

Reply:
![image](https://user-images.githubusercontent.com/2947151/147747240-8cfe3fb6-9a95-421a-b798-6d5ea20def73.png)